websites/M/Mavanimes/dist/metadata.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "Mavanimes",
+  "author": {
+    "name": "Adrien",
+    "id": "256458960666624003"
+  },
+  "category": "anime",
+  "logo": "https://i.imgur.com/PgicAOE.png",
+  "thumbnail": "https://i.imgur.com/XQddkkZ.png",
+  "version": "0.9.0",
```

new services should start at `1.0.0`
```suggestion
  "version": "1.0.0",
```